services/manifest-repo-export-service/pkg/db/cutoff_test.go

```diff
@@ -0,0 +1,113 @@
+/*This file is part of kuberpult.
+
+Kuberpult is free software: you can redistribute it and/or modify
+it under the terms of the Expat(MIT) License as published by
+the Free Software Foundation.
+
+Kuberpult is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+MIT License for more details.
+
+You should have received a copy of the MIT License
+along with kuberpult. If not, see <https://directory.fsf.org/wiki/License:Expat>.
+
+Copyright freiheit.com*/
+
+package cutoff
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"fmt"
+	"github.com/freiheit-com/kuberpult/pkg/db"
+	"github.com/freiheit-com/kuberpult/pkg/testutil"
+	"github.com/google/go-cmp/cmp"
+	"testing"
+)
+
+func TestTransformerWritesEslDataRoundTrip(t *testing.T) {
+	tcs := []struct {
+		Name  string
+		eslId db.EslId
+	}{
+		{
+			Name:  "1",
```

Comment:
Again, would appreciate more comprehensive test case names

Reply:
I extended the tests to have an Array of EslIds instead of just one.